src/sysusers/sysusers.c

```diff
@@ -1169,6 +1169,28 @@ static int add_user(Item *i) {
         return 0;
 }
 
+static int gid_is_available(gid_t gid) {
```

Comment:
hmm, so uid_is_ok() has this boolean parameter `check_with_gid`, which allows controlling whether the uid check should also imply a gid check. It appears we should do the inverse here, and thus modify gid_is_ok() to take a paramter `which_with_uid`.

Reply:
Ah good point, indeed that keeps it symmetrical. I guess parameter name suggestion was a typo for `check_with_uid`.

Reply:
yupp, it was. sorry for the confusion

Reply:
No problem, it was clearly understandable from the context. Reworked the patch that way.